reverted get started files -- header component

kayliezhao opened the pull request:
added header component & used for the how to get started
![image](https://github.com/user-attachments/assets/4ed00340-04c5-46a9-ac29-a764164fd18b)

kayliezhao committed:
header component

SafeDuck commented on src/app/home/gets-started.js:
```diff
@@ -0,0 +1,7 @@
+import GetStarted from "@/components/GetStarted";
+
+const Home = () => {
```
The folder and files shouldn't be created. Please remove. Next meeting will explain why with next.js routing system.

SafeDuck commented on src/components/Header.tsx:
```diff
@@ -1,7 +1,13 @@
+const Header = (props: { position: string; title: string }) => {
+  return (
+    <div
+      className={`w-1/8 flex flex-row border-b-4 border-t-4 border-white p-2 pl-[5%] pr-[5%] text-3xl text-white ${props.position === "left" ? "border-r-4" : "border-l-4"}`}
```
Are you sure `w-1/8` is a class in tailwind? 
Hover over it in vscode (tailwind intellisense)

SafeDuck replied:
Add font too.

kayliezhao committed:
fixed merge conflict

kayliezhao committed:
fixed font, deleted js file

kayliezhao commented:
deleted the extra file, changed font
<img width="426" alt="image" src="https://github.com/user-attachments/assets/95be2740-8180-44a4-9b98-37242911ce83">

SafeDuck commented on src/app/home/gets-started.js:
```diff
@@ -0,0 +1,7 @@
+import GetStarted from "@/components/GetStarted";
+
+const Home = () => {
+  return <GetStarted />;
```
delete this folder and files

SafeDuck commented on src/components/home/GetStarted.tsx:
```diff
@@ -1,7 +1,21 @@
+    <div className="flex w-full flex-col gap-3 text-3xl">
+      {/* change to justify-end when positioning to right */}
+      <div className="flex w-full justify-start">
+        <Header position="left" title={props.txt1} />
```
Revert this file back to it's original state

kayliezhao committed:
Merge branch 'dev' of https://github.com/acm-ucr/highlander-gloves-website into kayliezhao/header

kayliezhao committed:
reverted get started

kayliezhao commented:
reverted get started
